src/create.c

```diff
@@ -223,6 +224,7 @@ exec_create(int argc, char **argv)
 			{
 			const char *errstr;
 
+			level_is_set = true;
```

Comment:
I think this variable should be set to `true` merely if we have successfully parsed the `level`

Reply:
This is already done by:
```
			warnx("Invalid compression level %s", optarg);
			return (EXIT_FAILURE);
```